MITADS/corpus_api.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+python corpus_api.py && sort parlareitaliano_corpus_api.txt | uniq -u - > ./output/corpus_api.txt
```

Comment:
we have already a script that do the merge and sort in the repo, this one can be removed